src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -662,7 +662,7 @@ protected function getContainerBuilder()
     {
         $container = new ContainerBuilder(new ParameterBag($this->getKernelParameters()));
 
-        if (class_exists('ProxyManager\Configuration')) {
+        if (class_exists('ProxyManager\Configuration') && class_exists('Symfony\Bridge\ProxyManager\LazyProxy\Instantiator\RuntimeInstantiator')) {
```

Comment:
@aitboudad you are only looking at the case of installing subtree packages. When installing the fullstack repo, the RuntimeInstantiator class **always** exists, as it is part of the fullstack repo. But this does not mean that the ProxyManager library is installed